src/v3/src/transformer/terminal/transformTerminalTransaction.ts

```diff
@@ -84,6 +84,7 @@ const appendViewLinks = (
     type: 'Link',
     options: {
       label: loc('goback', 'login'),
+      step: 'cancel',
```

Comment:
I dont believe `cancel` is a step, it is an `action` right? So if it is passed in as a step wouldnt it be rejected with an error that this step does not exist?